restricted_site_access.php

```diff
@@ -109,49 +109,49 @@ public static function set_defaults( $blog_id, $user_id, $domain, $path, $site_i
 	 */
 	protected static function populate_fields_array() {
 		self::$fields = array(
-		'approach'      => array(
-			'default' => 1,
-			'label' 	=> __( 'Handle restricted visitors', 'restricted-site-access' ),
-			'field' 	=> 'settings_field_handling',
-		),
-		'message'       => array(
-			'default' => _x( 'Access to this site is restricted.', 'default restriction message', 'restricted-site-access' ),
-			'label' 	=> __( 'Restriction message', 'restricted-site-access' ),
-			'field' 	=> 'settings_field_message',
-		),
-		'redirect_url'  => array(
-			'default' => '',
-			'label'   => __( 'Redirect web address', 'restricted-site-access' ),
-			'field'   => 'settings_field_redirect',
-		),
-		'redirect_path'	=> array(
-			'default' => 0,
-			'label'   => 'Redirect to same path',
-			'field'   => 'settings_field_redirect_path',
-		),
-		'head_code'     => array(
-			'default' => 302,
-			'label'   => __( 'Redirection status code', 'restricted-site-access' ),
-			'field'   => 'settings_field_redirect_code',
-		),
-		'page'          => array(
-			'default' => 0,
-			'label' 	=> __( 'Restricted notice page', 'restricted-site-access' ),
-			'field'   => 'settings_field_rsa_page',
-		),
-		'allowed'       => array(
-			'default' => array(),
-			'label'   => __( 'Unrestricted IP addresses', 'restricted-site-access' ),
-			'field'   => 'settings_field_allowed',
-		),
+			'approach'      => array(
+				'default' => 1,
+				'label'   => esc_html__( 'Handle restricted visitors', 'restricted-site-access' ),
+				'field'   => 'settings_field_handling',
+			),
+			'message'       => array(
+				'default' => esc_html_x( 'Access to this site is restricted.', 'default restriction message', 'restricted-site-access' ),
+				'label'   => esc_html__( 'Restriction message', 'restricted-site-access' ),
+				'field'   => 'settings_field_message',
+			),
+			'redirect_url'  => array(
+				'default' => '',
+				'label'   => esc_html__( 'Redirect web address', 'restricted-site-access' ),
+				'field'   => 'settings_field_redirect',
+			),
+			'redirect_path' => array(
+				'default' => 0,
+				'label'   => 'Redirect to same path',
```

Comment:
this line is missing translation wrapper

Reply:
Done.